### ISSUE TITLE
eb create - ERROR: OS Error - [Errno 102]

### ISSUE THREAD

**Issue by iu-cedilla** [aws/aws-elastic-beanstalk-cli#18]
I got an error when creating a new environment using the CLI on my Mac. What's odd about this is the error message is saying related to operation not supported on socket related to Adobe Acrobat notification.

```bash
FOO-MBP01:~ foo$ eb create
Enter Environment Name
(default is foobar-01-dev): foobar-01-dev
Enter DNS CNAME prefix
(default is foobar-01-dev): foobar-01-dev

Select a load balancer type
1) classic
2) application
3) network
(default is 2): 2

Would you like to enable Spot Fleet requests for this environment?
(y/N): N
Creating application version archive "app-200304_175640".
ERROR: OSError - [Errno 102] Operation not supported on socket: './Library/Application Support/Adobe/Acrobat/DC/Reader/Synchronizer/Notification'
```

**Comment by halcarleton** [comment]
Thanks for reporting this. We will look into the issue and report back here when have an update.

**Comment by rahulrajaram** [comment]
@iu-cedilla , can you rerun `eb create` with the `--debug` argument? This will print a stack trace that will help identify the problem.

Also, which terminal software did you run this command on?

**Comment by halcarleton** [comment]
Closing this issue due to inactivity

**Comment by ahahn95** [comment]
@iu-cedilla I'm seeing this problem too, any luck?

**Comment by iu-cedilla** [comment]
No since I decided to not continue with this method. `eb create --debug` spits out a lot of info so you should capture it to a file.

**Comment by halcarleton** [comment]
This issue should be resolved in [v3.18.2](https://pypi.org/project/awsebcli/3.18.2/).
